dev/com.ibm.ws.st.liberty.buildplugin.integration/src/com/ibm/ws/st/liberty/buildplugin/integration/servertype/internal/AbstractLibertyBuildPluginJEEPublisher.java

```diff
@@ -107,7 +101,7 @@ public void prePublishApplication(int kind, PublishUnit app, MultiStatus status,
 
         if (wsServer != null) {
             String type = wsServer.getServerType();
-            if (Constants.SERVER_TYPE_LIBERTY_MAVEN.equals(type)) {
+            if (Constants.SERVER_TYPE_LIBERTY_MAVEN.equals(type) || Constants.SERVER_TYPE_LIBERTY_GRADLE.equals(type)) {
```

Comment:
For postPublishApplication it is overriding in the Maven specific code but for prePublishApplication it is not?